editor/plugins/animation_state_machine_editor.cpp

```diff
@@ -92,7 +92,7 @@ String AnimationNodeStateMachineEditor::_get_root_playback_path(String &r_node_d
 	if (edited_path.size()) {
 		while (!is_playable_anodesm_found) {
 			base_path = String("/").join(edited_path);
-			Ref<AnimationNodeStateMachine> anodesm = !edited_path.size() ? tree->get_tree_root() : tree->get_tree_root()->find_node_by_path(base_path);
+			Ref<AnimationNodeStateMachine> anodesm = !edited_path.size() ? Ref<AnimationNode>(tree->get_root_animation_node().ptr()) : tree->get_root_animation_node()->find_node_by_path(base_path);
```

Comment:
```suggestion
			Ref<AnimationNodeStateMachine> anodesm = !edited_path.size() ? tree->get_root_animation_node() : tree->get_root_animation_node()->find_node_by_path(base_path);
```
Why the weird conversion? Is it to silence some warning?

Reply:
Yes, get_root_animation_node() now only returns RootNode, so it will give a warning that the classes of the ternary operators do not match.